src/components/FlowRunCreateForm.vue

```diff
@@ -47,27 +46,34 @@
       </template>
 
       <p-accordion :sections="['Additional Options']">
+        <template #heading="{ section }">
```

Comment:
at some point I'd love to understand this syntax; I couldn't figure out in the flow runs accordion example how section / flowID / etc got unpacked